yellow-paper/docs/circuits/private-kernel-initial.md

```diff
@@ -22,8 +22,6 @@ For the _[function_data](#functiondata)_ in _[private_call](#privatecall).[call_
 
 - It must be a private function:
   - _`function_data.function_type == private`_
-- It must not be an internal function:
-  - _`function_data.is_internal == false`_
```

Comment:
Why removing this check?

Reply:
The plan is to replace it with an app-level check inside a macro: https://github.com/AztecProtocol/aztec-packages/issues/4449

Reply:
Good catch! Just removed them in [910f0fe](https://github.com/AztecProtocol/aztec-packages/pull/4432/commits/910f0feb9de03bae3fc495c7d7d538c7bab97b97)